test/common/schemaDynamicTest.js

```diff
@@ -20,12 +22,26 @@ var strings = typesRepresentatives.strings;
 
 var self;
 
+SchemaDynamicTest.TEST_STYLE = {
+	ASYNC: function (testFunction, argument, cb) {
+		testFunction(argument, cb);
+	},
+	THROWABLE: function (testFunction, argument, cb) {
+		try {
+			testFunction(argument);
+		} catch (ex) {
+			cb(ex);
+		}
+	}
+};
+
 function SchemaDynamicTest (config) {
 
 	this.customInput = config.customInput;
```

Comment:
We aren't using customInput anywhere, It would be great if you add the functionality for it.

Reply:
@MaciejBaj, can you add the necessary changes for customInput to work properly?